README.md

```diff
@@ -53,8 +53,27 @@ the open source community.
 
 ## Quick Start Guide
 
-The easiest way to deploy Volcano is to use the Helm chart.
+The easiest way to deploy Volcano is to use the Helm chart.  Volcano can be deployed by cloning code and also by adding helm repo.
 
+## Using Volcano Helm Repo
+## Helm Charts
```

Comment:
looks like an incorrect level or redundant header
```
## Helm Charts
```

Reply:
Have removed that header